cosmos/providers/dbt/core/operators.py

```diff
@@ -24,7 +26,7 @@ class DbtBaseOperator(BaseOperator):
     :param conn_id: The airflow connection to use as the target
     :type conn_id: str
     :param base_cmd: dbt sub-command to run (i.e ls, seed, run, test, etc.)
-    :type base_cmd: str
+    :type base_cmd: str or list
```

Comment:
I _think_ the type hint here should be:

```suggestion
    :type base_cmd: str | list[str]
```

Check out https://stackoverflow.com/a/58859298